lib/src/core/utils/extensions.dart

```diff
@@ -1,7 +1,23 @@
+import 'package:flutter/material.dart';
 import 'package:kana_to_kanji/src/core/constants/alphabets.dart';
 
 extension AlphabetsExtension on Alphabets {
   bool isKana() {
     return this == Alphabets.hiragana || this == Alphabets.katakana;
   }
 }
+
+extension ColorExtension on Color {
+  MaterialColor toMaterialColor() => MaterialColor(value, {
```

Comment:
A quoi ça correspond?

Reply:
Avec Material 3, il y a plein de nouvelle fonction qui change une couleur automatiquement en fonction du background, foreground, couleur voisines, etc...Du coup ils ont créé `MaterialColor` et `MaterialColorState`. Depuis, certaines fonction n'accepte pu de `Color` mais des `MaterialColor`, cette extension de `Color` va nous permettre de construire on the fly une `MaterialColor`.

Alors techniquement je pourrais le retirer puisque je l'utilisais et ne l'utilise plus, par contre ca reste utile c'est pour ca que je l'ai laissé. Je peux la retirer si tu prefere :D

Reply:
On peut le laisser, mais il faudrait laisser un commentaire pour expliquer à quoi ça correspond et potentiellement mettre un lien d'une doc externe si jamais y'en a une

Reply:
Je l'ai retirer, on verra si on en a besoin plus tard